portfolio/src/main/java/com/google/sps/servlets/DataServlet.java

```diff
@@ -29,11 +32,16 @@
 /** Servlet that returns some example content. TODO: modify this file to handle comments data */
 @WebServlet("/data")
 public class DataServlet extends HttpServlet {
-  
-  private List<String> messages = new ArrayList<String>();
 
   @Override
   public void doGet(HttpServletRequest request, HttpServletResponse response) throws IOException {
+    Query query = new Query("Comment");
+    DatastoreService datastore = DatastoreServiceFactory.getDatastoreService();
```

Comment:
Probably preferable to initialize this once for the servlet, instead of once per request. Typically, creation of a stub (like this) can be nontrivial in initialization time.